console/src/assets/i18n/it.json

```
@@ -776,6 +776,12 @@
       "ORGDETAIL_TITLE": "Inserisci il nome e il dominio della tua nuova organizzazione.",
       "ORGDETAIL_TITLE_WITHOUT_DOMAIN": "Inserisci il nome della tua nuova organizzazione.",
       "ORGDETAILUSER_TITLE": "Configurare il proprietario dell'organizzazione",
+      "RENAME": {
+        "ACTION": "Rinomina",
+        "TITLE": "Rinomina organizzazione",
```

Are titles also capitalized in italian?

```suggestion
        "TITLE": "Rinomina Organizzazione",
```

---

would keep this lowercase if you don't mind :D